test/__snapshots__/middleware.test.js.snap

```diff
@@ -0,0 +1,115 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`middleware logger should log error in "watch" method on startup: error 1`] = `
+Array [
+  Array [
+    [Error: Error in Watch method],
+  ],
+]
+`;
+
+exports[`middleware logger should log error in "watch" method on startup: info 1`] = `Array []`;
+
+exports[`middleware logger should log error in "watch" method on startup: warn 1`] = `Array []`;
+
+exports[`middleware logger should log on successfully build logs: error 1`] = `Array []`;
+
+exports[`middleware logger should log on successfully build logs: info 1`] = `
+Array [
+  Array [
+    "wait until bundle finished: /bundle.js",
+  ],
+  Array [
+    "Compiled successfully.",
+  ],
+  Array [
+    "Compiling...",
+  ],
+  Array [
+    "wait until bundle finished",
+  ],
+  Array [
+    "Compiled successfully.",
+  ],
+]
+`;
+
+exports[`middleware logger should log on successfully build logs: warn 1`] = `Array []`;
+
+exports[`middleware logger should log on unsuccessful build logs: error 1`] = `
+Array [
+  Array [
+    "
+ERROR in ./foo.js 1:3
+Module parse failed: Unexpected token (1:3)
+You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
+> 1()2()3()
+| ",
+  ],
+  Array [
+    "
+ERROR in ./foo.js 1:3
+Module parse failed: Unexpected token (1:3)
+You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
+> 1()2()3()
+| ",
+  ],
+]
+`;
+
+exports[`middleware logger should log on unsuccessful build logs: info 1`] = `
+Array [
+  Array [
+    "wait until bundle finished: /bundle.js",
+  ],
+  Array [
+    "Failed to compile.",
+  ],
+  Array [
+    "Compiling...",
+  ],
+  Array [
+    "wait until bundle finished",
+  ],
+  Array [
+    "Failed to compile.",
+  ],
+]
+`;
+
+exports[`middleware logger should log on unsuccessful build logs: warn 1`] = `Array []`;
+
+exports[`middleware logger should log on warning build logs: error 1`] = `Array []`;
+
+exports[`middleware logger should log on warning build logs: info 1`] = `
+Array [
+  Array [
+    "wait until bundle finished: /bundle.js",
+  ],
+  Array [
+    "Compiled with warnings.",
+  ],
+  Array [
+    "Compiling...",
+  ],
+  Array [
+    "wait until bundle finished",
+  ],
+  Array [
+    "Compiled with warnings.",
+  ],
+]
+`;
+
+exports[`middleware logger should log on warning build logs: warn 1`] = `
+Array [
+  Array [
+    "
+WARNING in Warning",
```

Comment:
"WARNING in Warning" 🤔?

Reply:
It it just test https://github.com/webpack/webpack-dev-middleware/pull/494/files#diff-ea1fcb38e70938ae981c4e16e60903a0R16 :smile: because webpack can emit warning as error, so i want to keep what is this in message